.changeset/true-poets-switch.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.webhooks.v1": patch
+"@wso2is/console": patch
+---
+
+The action bar was momentarily visible during the Webhooks page suspense/loading state,hence Updated `showTopActionPanel` to render only after the loading state completes
```

Comment:
Let's not add implementation details in the changelog. Just adding a brief summary of the change is enough. 

```suggestion
Fix Action bar is momentarily visible during the Webhooks page suspense state
```